packages/contractkit/src/utils/ber-utils.ts

```diff
@@ -0,0 +1,54 @@
+import * as asn1 from 'asn1js'
+import { BigNumber } from 'bignumber.js'
+import { bigNumberToBuffer, bufferToBigNumber } from './signature-utils'
+
+export const toArrayBuffer = (b: Buffer): ArrayBuffer => {
+  return b.buffer.slice(b.byteOffset, b.byteOffset + b.byteLength)
+}
+
+export function publicKeyFromAsn1(b: Buffer): BigNumber {
```

Comment:
are these functions pretty generic or specific to expected responses from Azure/AWS?

Reply:
This method takes a BER encoded key with ASN1 serialization protocol and converts it to a BigNumber format, which our libraries are using as the standard. AWS sends the key in this format so although this method is generic, I don't expect other libraries to use this.